flutter-studio/src/io/flutter/actions/FlutterNewProjectAction.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2017 The Chromium Authors. All rights reserved.
+ * Use of this source code is governed by a BSD-style license that can be
+ * found in the LICENSE file.
+ */
+package io.flutter.actions;
+
+import com.intellij.ide.impl.NewProjectUtil;
+import com.intellij.ide.projectWizard.NewProjectWizard;
+import com.intellij.ide.projectWizard.ProjectTypeStep;
+import com.intellij.ide.util.projectWizard.ModuleWizardStep;
+import com.intellij.openapi.actionSystem.AnAction;
+import com.intellij.openapi.actionSystem.AnActionEvent;
+import com.intellij.openapi.roots.ui.configuration.ModulesProvider;
+import com.intellij.ui.CollectionListModel;
+import com.intellij.ui.components.JBList;
+import com.intellij.util.ReflectionUtil;
+
+public class FlutterNewProjectAction extends AnAction {
```

Comment:
I think we should add a TODO in this class that we'll eventually want to move to a mechanism that just shows our wizard, w/o a list on the left.